eslint-rules/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@desmodder/eslint-rules",
+  "version": "0.1.0",
+  "private": true,
+  "type": "module",
+  "description": "Local ESLint rules for DesModder development",
+  "main": "./dist/index.js",
+  "exports": {
+    ".": {
+      "import": {
+        "types": "./dist-ts/index.d.ts",
+        "default": "./dist/index.js"
+      }
+    }
+  },
+  "scripts": {
+    "build": "tsc --build && node esbuild.mjs",
+    "postinstall": "npm run build"
```

Comment:
Nice, so this'll automatically build on `npm install`. I was worried there would be an extra dev step.